flow/Makefile

```diff
@@ -743,7 +743,7 @@ $(GDS_MERGED_FILE): $(RESULTS_DIR)/6_final.def $(OBJECTS_DIR)/klayout.lyt $(GDSO
 $(RESULTS_DIR)/6_final.v: $(LOG_DIR)/6_report.log
 
 $(GDS_FINAL_FILE): $(GDS_MERGED_FILE)
-	cp $< $@
+	ln -sf $(shell realpath --relative-to=$(dir $@) $<) $@
```

Comment:
@maliberty The relative path is relative to the destination folder of the symlink location.